js/components/pages/PassagePage.react.js

```diff
@@ -20,6 +21,9 @@ class PassagePage extends Component {
       previousButton = <button className="previous" onClick={() => { dispatch(asyncPreviousVerse()) }}>Previous</button>
     }
 
+    const audioUrl = "http://www.esvapi.org/v2/rest/passageQuery?key=IP&passage=" +
+      verseMetadata.replace(/ /, "+") + "&output-format=mp3";
```

Comment:
Could simplify this encoding with with `encodeUri` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI

```js
const audioUrl = encodeUri("http://www.esvapi.org/v2/rest/passageQuery?key=IP&passage=" + verseMetadata + "&output-format=mp3");
```